README.md

```diff
@@ -96,6 +96,125 @@ Detailed **[documentation](https://werf.io/documentation/)** is available in mul
+- The description of all rules for building and deploying an application (that may have any number of components) is stored in a single Git repository along with the source code (Single Source Of Truth).
+- Build images using Dockerfiles.
+- Alternatively, werf provides a custom builder tool with support for custom syntax, Ansible, and incremental rebuilds based on Git history.
+- werf supports Helm compatible charts and complex fault-tolerant deployment processes with logging, tracking, early error detection, and annotations to customize the tracking logic of specific resources.
+- werf is a CLI tool written in Go. It can be embedded into any existing CI/CD system to implement CI/CD for your application.
+- Cross-platform development: Linux-based containers can be run on Linux, macOS, and Windows.
+
+### Building
+
+- Effortlessly build as many images as you like in one project.
+- Build images using Dockerfiles or Stapel builder instructions.
+- Build images concurrently on a single host (using file locks).
+- Build images simultaneously.
+- Build images distributedly.
+- Content-based tagging.
+- Advanced building process with Stapel:
+  - Incremental rebuilds based on git history.
+  - Build images with Ansible tasks or Shell scripts.
+  - Share a common cache between builds using mounts.
+  - Reduce image size by detaching source data and building tools.
+- Build one image on top of another based on the same config.
+- Debugging tools for inspecting the build process.
+
+### Deploying
+
+- Deploy an application to Kubernetes and check if it has been deployed correctly.
+  - Track the statuses of all application resources.
+  - Control the readiness of resources.
+  - Control the deployment process with annotations.
+- Full visibility of both the deployment process and the final result.
+  - Logging and error reporting.
+  - Regular status reporting during the deployment phase.
+  - Debug problems effortlessly without unnecessary kubectl invocations.
+- Prompt CI pipeline failure in case of a problem (i.e. fail fast).
+  - Instant detection of resource failures during the deployment process without having to wait for a timeout.
+- Full compatibility with Helm.
+
+### Cleaning up
+
+- Clean up container registry by enforcing customizable policies.
+- Keep images that are being used in the Kubernetes cluster. werf scans the following kinds of objects: Pod, Deployment, ReplicaSet, StatefulSet, DaemonSet, Job, CronJob, ReplicationController.
+
+## Production ready
+
+werf is a mature, reliable tool you can trust. [Read about release channels](https://werf.io/about/release_channels.html).
+
+## Documentation
+
+Detailed **[documentation](https://werf.io/documentation/)** is available in multiple languages.
+
+**[Many guides](https://werf.io/guides.html)** are provided for developers to quickly deploy apps to Kubernetes using werf.
+
+
```

Comment:
Bad merge. See how it looks here https://github.com/werf/werf/blob/daf5e8f7251c62d946b59fa998765e241bf03960/README.md.

Reply:
whoops